browser.js

```diff
@@ -7,8 +7,8 @@ function getDown () {
   return leveljs
 }
 
-function getLocation (name) {
-  return name
+function getLocation () {
+  return 'db_' + Date.now()
```

Comment:
Maybe we should add a sequence here, or use something like https://github.com/dominictarr/monotonic-timestamp

Reply:
@vweevers Hmm, WDYT about using guids instead? It would be nice if the database names were always random, regardless if disk-based or not.

Reply:
I like that